README.md

```diff
@@ -25,29 +25,18 @@ Install this package as any other Python packages:
 $ pip install -U chaostoolkit-reporting
 ```
 
-Notice that this draws a few [dependencies][deps]:
+Notice that this draws a few [dependencies][deps] that will need to be installed:
 
 [deps]: https://github.com/chaostoolkit/chaostoolkit-reporting/blob/master/requirements.txt
 
+```
+$ pip install -U -r requirements.txt
+```
+
```

Comment:
I think this might have been just misinterpreted;
`Notice that this draws a few [dependencies][deps]:` is referring to the dependencies that are brought in by installing `chaostoolkit-reporting`. The purpose of having this statement is just to alert users to the fact that the dependencies of `chaostoolkit-reporting` have `LGPL V3` licenses, which some companies/folks aren't too fond of.

I think we can leave this as it was, it's more a warning than a 'you need to install the requirements in this repo'

Reply:
Ah right, gotcha. No need for this then